### PR TITLE
Update plugin configuration

### PR DESCRIPTION
**Pull request by prabushi**
## Purpose
Fix https://github.com/ballerina-platform/plugin-vscode/issues/204

Screenshot 2021-02-11 at 11 42 11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,57 +65,52 @@
...
                 "ballerina.dataMapper.url": {
                     "type": "string",
                     "default": "https://datamapper.choreo.dev",
-                    "description": "URL of the host for data mapper back end"
+                    "description": "Specify the URL of the data mapper backend host."
+                },
+                "ballerina.goToDefinition.enableStandardLibraries": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Enable navigation into standard library content with goto definition feature.\n(Supports Ballerina 1.2.xx versions)"
```
Comment (rasika) [inline package.json:113]: Let's reword `1.2.xx` into `1.2.x`

File: package.json [hunk 2]
```diff
@@ -65,57 +65,52 @@
...
                     "type": "string",
                     "default": "",
-                    "description": "Specifies the path to the ballerina home directory"
+                    "description": "Specify the Ballerina home directory path."
                 },
                 "ballerina.allowExperimental": {
                     "type": "boolean",
                     "default": false,
-                    "description": "If set to true experimental ballerina features will be allowed"
+                    "description": "Enable experimental features.\n(Supports Ballerina 1.2.xx versions)"
```
Comment (rasika) [inline package.json:78]: Let's reword `1.2.xx` into `1.2.x`

File: package.json (hunk 1 above)
Comment (prabushi) [reply package.json]: Updated.

File: package.json (hunk 2 above)
Comment (prabushi) [reply package.json]: Updated.

File: package.json [hunk 3]
```diff
@@ -65,57 +65,52 @@
...
-                    "type": "boolean",
-                    "default": true,
-                    "description": "If set to true ballerina endpoints code lens feature will be allowed"
+                    "description": "Enable documentation code lens feature.\n(Supports Ballerina Swan Lake)"
                 },
                 "ballerina.dataMapper.enabled": {
                     "type": "boolean",
                     "default": false,
-                    "description": "If set to true ballerina data mapper code action feature will be allowed"
+                    "description": "Enable data mapper code action feature."
```
Comment (nadeeshaan) [inline package.json:103]: Do we need to add the word `feature` here?

Comment (prabushi) [reply package.json]: Updated.

**Review by nadeeshaan — APPROVED** [review]

**Pull request merged**